next/app/projects/projects.ts

```diff
@@ -13,7 +13,7 @@ export interface Project{
 
 export const projectsData: Project[] = [
     {
-        logo: defaultImage,
+        logo: "/images/projects-1.gif",
```

Comment:
This gif isn't sizing up to be the same as the others, hopefully we can have all of them be the same size!